core/audits/redirects.js

```diff
@@ -125,9 +125,10 @@ class Redirects extends Audit {
       }
 
       const lanternTimingDeltaMs = redirectedTiming.startTime - initialTiming.startTime;
-      const observedTimingDeltaS = redirectedRequest.startTime - initialRequest.startTime;
+      const observedTimingDelta =
+        redirectedRequest.mainThreadStartTime - initialRequest.mainThreadStartTime;
```

Comment:
the redirect's `mainThreadStartTime` is the same as requestWillBeSent timestamp. The name made sense for the original request but I suppose it's badly named for the redirected request, given the main thread is not involved in handling 3xx. Do you have a suggestion here, or maybe just a comment should be added on this line?

Reply:
In a followup refactor I'm interested in having redirects belong to the single request. (Instead of this weird cloned multi-request thing). I don't totally grok this discussion here, but it feels like the semantics are currently awkward.

Reply:
> but it feels like the semantics are currently awkward.

yeah, that's the main issue, created timestamps that don't really fit the naming of a regular request because it's not a real request. Followup refactor sounds good.